app/src/main/java/org/novinger/jason/demoreader/datamodels/Article.java

```diff
@@ -0,0 +1,67 @@
+package org.novinger.jason.demoreader.datamodels;
+
+/**
+ * Created by jason on 4/27/16.
+ */
+public class Article {
+    private String id;
```

Comment:
so the convention here is private members use the `m` prefix.  `mId`.  this may seem wierd, but it does allow for easy getter/setter generation...

```
private int mId;
public void setId(int id){
  mId = id;
}
```

What's nice about that too is that the private `mId` won't be in Javadocs, but the setter will, which will describe it's argument as `id` (so the user only sees the sane reading `id`)